presto-main/src/main/java/com/facebook/presto/type/DecimalInequalityOperators.java

```diff
@@ -157,9 +157,7 @@ private static boolean rescaledValuesFitJavaLong(SpecializeContext context)
     {
```

Comment:
this function is inconsistent with `shortRescaleExtraParameters`. Please unify it.